package.json

```diff
@@ -35,7 +35,7 @@
     "precompile": "gts clean"
   },
   "dependencies": {
-    "@deliveryhero/google-auth-library": "^7.6.1",
+    "@deliveryhero/google-auth-library": "git+ssh://git@github.com/deliveryhero/google-auth-library-nodejs.git#DAT-8016",
```

Comment:
Let's maybe publish it as `@deliveryhero/google-auth-library`? Otherwise host repository will have to compile it